tests/css_trailing_comma/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -11,16 +11,64 @@ a {
 
 `;
 
+exports[`declaration.css 2`] = `
+a {
+    margin: $bar,;
+}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+a {
+  margin: $bar;
```

Comment:
What is this test trying to say? Are you specifically trying to remove that trailing comma, or is this just an observation you made? I guess it doesn’t really matter what happens to the comma in this case. Just curious.

Reply:
@lydell here the trailing comma should not be shown as it was previously in `line 9`. I had just added `trailingComma: "es5" ` in `jsfmt.spec.js` so this is showing here as all the tests are again run for `trailingComma: "es5" `